config/samples/saas_v1alpha1_shardedredisbackup.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: saas.3scale.net/v1alpha1
+kind: ShardedRedisBackup
+metadata:
+  labels:
+    app.kubernetes.io/name: shardedredisbackup
+    app.kubernetes.io/instance: shardedredisbackup-sample
+    app.kubernetes.io/part-of: saas-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: saas-operator
+  name: shardedredisbackup-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Can you maybe add to this sample the content of the PR description where there is a good example? including the finalizer, since it is important?

Reply:
the finalizer is automatically set by the controller, but ok to add a sample here

Reply:
added in a09ae99